dbms/src/TestUtils/ExecutorTestUtils.cpp

```diff
@@ -104,6 +104,38 @@ void ExecutorTest::executeInterpreter(const String & expected_string, const std:
     ASSERT_EQ(Poco::trim(expected_string), Poco::trim(fb.toString()));
 }
 
+void ExecutorTest::executeExecutor(
+    const std::shared_ptr<tipb::DAGRequest> & request,
+    std::function<void(const ColumnsWithTypeAndName &)> assert_func)
+{
+    WRAP_FOR_DIS_ENABLE_PLANNER_BEGIN
+    std::vector<size_t> concurrencies{1, 2, 10};
```

Comment:
Maybe we should provide some  ways to set the concurrency policy, such as number of  concurrencies or degree of concurrency, by users.

Reply:
I think they can use `executeStream(dagrequest, concurrency)` directly.
Specifying a specific degree of concurrency is a rare requirement.